stdlib/public/RemoteInspection/TypeLowering.cpp

```diff
@@ -2103,44 +2120,11 @@ class EnumTypeInfoBuilder {
       }
     }
 
-    // See if we have MPE bit mask information from the compiler...
```

Comment:
Previously, I tried having the compiler export a "multi-payload enum descriptor" with a complete copy of the spare bit mask.  I was never happy with this approach because it potentially adds a lot of bulk to the reflection metadata.  Fortunately, the new code has shown we shouldn't need such information from the compiler and this PR rips all of that logic out of RemoteMirror.  I'll file a separate PR to remove the never-completely-implemented compiler support for generating this data.